src/main/java/overclocked/stl/io/OverclockedController.java

```diff
@@ -0,0 +1,30 @@
+
+package overclocked.stl.io;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+
+/** 
+ * An extension of the {@link GenericHID} class that adds some useful utilities.
+ * */
+public class OverclockedController extends GenericHID {
+
+  /**
+   * Creates a new OverclockedController.
+   *
+   * @param port The port index on the Driver Station that the device is plugged into.
+   */
+  public OverclockedController(int port) {
+    super(port);
+  }
+
+  /**
+   * Creates a new {@link JoystickButton} from this controller.
```

Comment:
Maybe it should say “new JoystickButton ‘for’ this controller.”?

Reply:
The "from" is intentional, as the controller itself is an input into the JoystickButton constructor